### PR TITLE
add multi-site support from ceph-ansible task

### PR DESCRIPTION
**Pull request by sunilkumarn417**
- Code changes to support RGW multi-site from ceph-ansible task.
- Added multi-site ceph-ansible task for single realm.
- Multi-site and single site configuration tested locally.

Signed-off-by: sunilkumarn417 <sunnagar@redhat.com>

### CODE REVIEW

File: qa/suites/system-tests/object/multi-site/clusters/single_realm_cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+overrides:
```
Comment (rakeshgm) [inline qa/suites/system-tests/object/multi-site/clusters/single_realm_cluster.yaml]: I think moving this file to https://github.com/red-hat-storage/ceph/tree/master/qa/suites/system-tests/clusters makes more sense and create a symlink for it in multisite dir.

Comment (sunilkumarn417) [reply qa/suites/system-tests/object/multi-site/clusters/single_realm_cluster.yaml]: updated.

**Comment by sunilkumarn417** [comment]
http://pulpito.ceph.redhat.com/sunnagar-2020-05-12_15:17:42-system-tests:object:multi-site-rh-nautilus-distro-basic-clara/

failure are not related to ceph-ansible task.

**Comment by sunilkumarn417** [comment]
closing this PR, as this commit is submitted to master

**Pull request closed without merging**